srcpkgs/gloobus-preview/template

```diff
@@ -23,5 +23,6 @@ if [ -z "$CROSS_BUILD" ]; then
 fi
 
 pre_configure() {
-	autoreconf -fi
+	sed -i '/ACX_PTHREAD()/d' configure.ac
+	autoreconf -fiv
```

Comment:
remove -v

Reply:
Weirdly enough now it is compiling even without the sed command... 

`./xbps-src clean && ./xbps-src pkg ` gives me a clean build state right ?

Reply:
mostly, clean only cleans directories in /builddir and /destdir/. If one of the previous builds touched the system outside of these directories (and you were not using -t), it would still be broken. `zap` and `binary-bootstrap` recreate the masterdir from scratch, so that definitely gives you a clean state. `clean` should be good enough for all packages.

Reply:
Tried building now this package after a zap 

could you confirm that this package builds without the sed ?